.github/workflows/rust.yml

```diff
@@ -11,12 +11,22 @@ env:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v3
-    - name: Build
-      run: cargo +stable build --verbose
-    - name: Run tests
-      run: cargo +stable test --verbose
+      - uses: actions/checkout@v2
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: nightly
+            override: true
+            components: rustc
```

Comment:
We should open an issue to get back on stable if we haven't already

Reply:
We have, but I need to bring over all the issues from the other repos.

Reply:
#29